core/databases/db_completion_tasks.py

```diff
@@ -37,6 +37,12 @@ def db_get_completion_tasks_by_page(page: int, per_page: int = defaults.ITEMS_PE
     return results
 
 
+def db_get_completion_tasks_by_uuid(uuid: int):
+    fields = Query()
+    result = db.get(fields.uuid == uuid)
+    return result
+
```

Comment:
Plural `tasks` implies a list is being returned, but you're using `.get` which returns only a single result.
Please fix.